Library/ProjectPamphlet+Helpers.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+import KsApi
+
+public func userCanSeeNativeCheckout() -> Bool {
+  return nativeCheckoutExperimentIsEnabled() && featureNativeCheckoutEnabled()
```

Comment:
I completely missed the naming the first time around...

What do you say we make this more consistent so that we:
* use `isEnabled` for both
* use the same pattern (either: `nativeCheckoutExperimentIsEnabled` & `nativeCheckoutFeatureIsEnabled` or `experimentNativeCheckoutIsEnabled` & `featureNativeCheckoutIsEnabled`)?